docs/Content/HideSbieProcesses.md

```diff
@@ -0,0 +1,15 @@
+# Hide Sandboxie Processes
+
+HideSbieProcesses is a sandbox setting in [Sandboxie Ini](SandboxieIni.md).
+
+```
+   .
+   .
+   .
+   [DefaultBox]
+   HideSbieProcesses=y
+```
+
+Use the 'HideSbieProcesses=y' option to hide Sandboxie Work Process (SbieSvc, SandboxieRpcSs etc.)
```

Comment:
```suggestion
Use the 'HideSbieProcesses=y' option to hide Sandboxie worker processes (SbieSvc, SandboxieRpcSs etc.)
```

For this line, I would like to ask for extra confirmation from @offhub, because if confirmed, it may require a fix in the changelog.

Reply:
Probably so, as changelog says as my text suggests.
FYI I have made some screenshots as follows(standard non-admin with `HideNonSystemProcesses=y`):
with `HideSbieProcesses=n`:
<img width="183" alt="image" src="https://github.com/user-attachments/assets/5d99e3c9-6f2c-46e5-8a38-0a2940a00648">
and with `HideSbieProcesses=y`:
<img width="178" alt="image" src="https://github.com/user-attachments/assets/deab208d-c1d9-42f4-b84d-88de09a77ba8">

Reply:
Setting `HideSbieProcesses=y` hides processes with image names that start with 'Sandboxie' or 'Sbie.' [code](https://github.com/sandboxie-plus/Sandboxie/blob/54b86d587f1b2829ff9d550315ea5aa814b92ec3/Sandboxie/core/dll/sysinfo.c#L414-L416)

Reply:
> For this line, I would like to ask for extra confirmation from @offhub, because if confirmed, it may require a fix in the changelog.

@DavidXanatos

Reply:
https://github.com/sandboxie-plus/sandboxie-docs/pull/187#discussion_r1765426608

@DavidXanatos